imagestreams/ruby-centos7.json

```diff
@@ -29,40 +29,40 @@
         }
       },
       {
-        "name": "2.5",
+        "name": "2.6",
         "annotations": {
-          "openshift.io/display-name": "Ruby 2.5",
+          "openshift.io/display-name": "Ruby 2.6",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Build and run Ruby 2.5 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/blob/master/2.5/README.md.",
+          "description": "Build and run Ruby 2.6 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/blob/master/2.6/README.md.",
           "iconClass": "icon-ruby",
-          "tags": "builder,ruby",
-          "supports": "ruby:2.5,ruby",
-          "version": "2.5",
+          "tags": "hidden,builder,ruby",
```

Comment:
I dont think this should be hidden anymore. I expect it was only hidden because the centos7 image did not exist yet? @hhorak

Reply:
I'm not sure to be honest, but @pkubatrh's explanation makes sense and we don't have it in other images, so I'm +1 for removing the hidden label here.

Reply:
I have removed it.